apps/web-client/src/lib/forms/WarehouseForm.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+	import type { ZodValidation, SuperValidated } from "sveltekit-superforms";
+	import type { SuperForm } from "sveltekit-superforms/client";
+	import { superForm, superValidateSync, numberProxy } from "sveltekit-superforms/client";
+
+	import { warehouseSchema, type WarehouseFormData } from "$lib/forms/schemas";
+
+	import { Input } from "$lib/components/FormControls";
+	import { Percent } from "lucide-svelte";
+
+	type Form = SuperForm<ZodValidation<typeof warehouseSchema>, unknown>;
+
+	export let data: WarehouseFormData;
+	export let options: Form["options"];
+	/**
+	 * Handle click of "X" icon button
+	 */
+	export let onCancel: (e: Event) => void = () => {};
+
+	const form = superForm(superValidateSync(data, warehouseSchema), options);
+
+	const { form: formStore, constraints, enhance } = form;
+
+	const discountProxy = numberProxy(formStore, "discount", { emptyIfZero: false, empty: "undefined" });
+</script>
+
+<form class="flex max-w-lg flex-col gap-y-6" aria-label="Edit warehouse details" use:enhance method="POST">
+	<div class="flex flex-col justify-between gap-y-6 p-6">
+		<input type="hidden" name="id" value={$formStore.id} />
+		<div class="basis-full">
+			<Input bind:value={$formStore.name} name="name" label="Name" placeholder="Dish " {...$constraints.name} />
+		</div>
```

Comment:
This is cool if `bind:value` works, but I'd really prefere the naked `<input>` element with style classes, rather than a full on component (if possible ofc)

Reply:
If I remember correctly, the two-way bind was a problem with multiple choice checkboxes, so...since we don't have those here, not really necessary (just a preference)